plugin/DataView/DataDetailViewHandler.php

```diff
@@ -42,13 +42,17 @@ class DataDetailViewHandler
 	/** @var WPOptionWrapperBase */
 	private $_pWPOptionWrapper;
 
+    /** @var  RecordManagerPostMeta */
+    private $_pRecordPostMeta;
+
 
 	/**
 	 * @param WPOptionWrapperBase $pWPOptionWrapper
 	 */
 	public function __construct(WPOptionWrapperBase $pWPOptionWrapper = null)
 	{
 		$this->_pWPOptionWrapper = $pWPOptionWrapper ?? new WPOptionWrapperDefault();
+        $this->_pRecordPostMeta = new RecordManagerPostMeta();
```

Comment:
`$this->_pRecordPostMeta = $pRecordPostMeta ?? new RecordManagerPostMeta()` would be better here (`$pRecordPostMeta` being a constructor argument), so that all `new DataDetailViewHandler` can quickly be replaced with `$pContainer->get(DataDetailViewHandler::class)`.

Reply:
@jayay , i fixed it, can you check it ?